apps/etterlatte-brev-api/src/main/kotlin/no/nav/etterlatte/brev/VedtaksbrevService.kt

```diff
@@ -85,15 +88,22 @@ class VedtaksbrevService(
     }
 
     private suspend fun opprettEllerOppdater(behandling: Behandling): UlagretBrev {
-        val enhet = behandling.vedtak.enhet
-        val avsender = adresseService.hentAvsenderEnhet(enhet)
+        val saksbehandlerEnhet = behandling.vedtak.saksbehandlerEnhet
+        val saksbehandlerIdent = behandling.vedtak.saksbehandlerIdent
+        val saksbehandlerInfo = navansattKlient.hentSaksbehandlerInfo(saksbehandlerIdent)
+
+        val attestantEnhet = behandling.vedtak.attestantEnhet?.let { adresseService.hentEnhet(it) }
+        val attestantInfo = behandling.vedtak.attestantIdent?.let { navansattKlient.hentSaksbehandlerInfo(it) }
+        val attestant = Attestant(attestantInfo?.navn ?: "", attestantEnhet ?: "")
```

Comment:
Dette virker ikke helt heldig -- dette virker for meg som en nullable attestant i stedet, og så må man evt. håndtere at et brev ikke har attestant før man er ferdig med fatting av vedtak og faktisk attestering.

Mulig jeg tolker flyten veldig pessimistisk her, menn nedetid i navansatt-tjenesten (eller en bug i kallene opp mot de) vil produsere brev som bare mangler navn -- uten at man stopper eksplisitt opp en plass? Må attestant fange dette før de trykker på attester?

Reply:
Ved feil i kallene mot f.eks navansatt eller norg2 så vil ikke brevet bli generert, så som saksbehandler/attestant vil du ikke kunne få opp brevet og dermed ikke kunne se det før de iverksetter / sender det til attestering. Er ikke en helt heldig løsning, men vil fungere i første omgang. 

Attestant må uansett lese brevet gjennom for å dobbeltsjekke verdiene som ligger inne, så de burde se om noe mangler?